### PR TITLE
docs: help copy-pasters with configuration

### PR DESCRIPTION
**Pull request by szepeviktor**
I see many developers using this very configuration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,18 +52,18 @@ includes:
...
-        - ./*/*/FileToBeExcluded.php
-
-    checkMissingIterableValueType: false
+#    ignoreErrors:
+#        - '#PHPDoc tag @var#'
+#
+#    excludePaths:
+#        - ./*/*/FileToBeExcluded.php
+#
+#    checkMissingIterableValueType: false
```
Comment (nunomaduro) [inline README.md:66]: Feel that, for most Laravel applications, we should start with `checkMissingIterableValueType: false`. @canvural What's your thoughts on this one?

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (szepeviktor) [reply README.md:66]: > we should start with `checkMissingIterableValueType: fals`

That starts only on level 6, the above example has level 5.

**Review by nunomaduro — APPROVED** [review]
Going to let @canvural validate and merge this one.

File: README.md (hunk 1 above)
Comment (canvural) [reply README.md:66]: I think I agree with both of you 😄 As @szepeviktor said that check only applies for level 6 and above. So leaving it commented here would be a good idea, if anyone wants to go above level 6.

**Comment by canvural** [comment]
Thank you!

**Pull request merged**